brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced columns - Column not stacking on tablet & mobile devices.

### Description

Trello - https://trello.com/c/a0Bsp9IW/255-fix-advanced-columns-column-not-stacking-on-tablet-mobile-devices

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop advanced column with 2 child with 50% width.
- Select option stack on > tablet and check is stacking on tablet and mobile both.
- Select option stack on > mobile. It should stack on mobile devices only.
- Select option stack on > none.  It should not stack on tablet & mobile devices.  

### Checklist:
